components/EditionMenu.vue

```diff
@@ -35,12 +35,19 @@
 <template>
     <div>
         <div class="icon-container">
-            <div class="edit-icon icon" @click="$emit('deleteEvent')">delete</div>
-            <div class="edit-icon icon" @click="$emit('frontEvent')">flip_to_front</div>
-            <div class="edit-icon icon" @click="$emit('backEvent')">flip_to_back</div>
-            <div class="edit-icon icon" @click="$emit('mirrorHorizontalEvent')">flip</div>
-            <div class="edit-icon--flipped edit-icon icon" @click="$emit('mirrorVerticalEvent')">flip</div>
-            <div class="edit-icon icon" @click="handleTextAlignSwitch" v-if="props.element.type.name === 'Text'">
+            <div class="edit-icon icon" @click="$emit('deleteEvent')" title="Delete">delete</div>
+            <div class="edit-icon icon" @click="$emit('frontEvent')" title="Move Forward">flip_to_front</div>
+            <div class="edit-icon icon" @click="$emit('backEvent')" title="Move Backward">flip_to_back</div>
```

Comment:
I think maybe we should also clarify here that it is only moving one step to the front